apps/services/auth/admin-api/src/app/v2/clients/test/me-clients.spec.ts

```diff
@@ -1,11 +1,190 @@
+import faker from 'faker'
+import request from 'supertest'
+
+import { AdminPortalScope } from '@island.is/auth/scopes'
+import {
+  GrantTypeEnum,
+  SequelizeConfigService,
+  TranslatedValueDto,
+} from '@island.is/auth-api-lib'
+import { User } from '@island.is/auth-nest-tools'
+import { FixtureFactory } from '@island.is/services/auth/testing'
+import { createCurrentUser } from '@island.is/testing/fixtures'
+import { getRequestMethod, setupApp, TestApp } from '@island.is/testing/nest'
+
+import { AppModule } from '../../../app.module'
+
+const tenantId = '@test.is'
+const clientId = '@test.is/test-client'
+
+const createTestClientData = async (app: TestApp, user: User) => {
+  const fixtureFactory = new FixtureFactory(app)
+  await fixtureFactory.createDomain({
+    name: tenantId,
+    nationalId: user.nationalId,
+  })
+  const client = await fixtureFactory.createClient({
+    clientId: clientId,
+    domainName: tenantId,
+  })
+  const redirectUri = await fixtureFactory.createClientRedirectUri({ clientId })
+  const postLogoutRedirectUri = await fixtureFactory.createClientPostLogoutRedirectUri(
+    {
+      clientId,
+    },
+  )
+  await fixtureFactory.createClientGrantType({
+    clientId,
+    grantType: GrantTypeEnum.TokenExchange,
+  })
+  const [translation] = await fixtureFactory.createTranslations(client, 'en', {
+    clientName: faker.random.word(),
+  })
+  const clientClaim = await fixtureFactory.createClientClaim({
+    clientId,
+    type: faker.random.word(),
+    value: faker.random.word(),
+  })
```

Comment:
~I like supporting related models like these in the `CreateClient` interface. Like we do with api scopes in createDomain and createApiScopeGroup.